### ISSUE TITLE
Netcdf3 support

### ISSUE THREAD

**Issue by jbusecke** [jbusecke/esgf-virtual-zarr-data-access#2]
Do we expect/plan to support netcdf3 for CMIP6+/CMIP7?

If the answer is yes, we need to track and support https://github.com/TomNicholas/VirtualiZarr/issues/86

**Comment by TomNicholas** [comment]
I don't think you need to follow https://github.com/TomNicholas/VirtualiZarr/issues/86 - we will support generating references from netCDF3 files regardless. That issue is more about whether we can do something clever with the chunking in the case of uncompressed data.

**Comment by TomNicholas** [comment]
FYI virtualizarr has supported generating references from netCDF3 files for a while now.

**Comment by jbusecke** [comment]
I also recently learned that netcdf3 is not supported in CMIP6....but I guess some modeling centers got around the quality control 🙈
